<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetIOslots</class>
 <widget class="QWidget" name="WidgetIOslots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="tvIoslots">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>411</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>591</width>
     <height>411</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>589</width>
      <height>409</height>
     </rect>
    </property>
    <widget class="QToolButton" name="tbSetSlot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string>Set slot</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/magic_wand_2.png</normaloff>:/icons/magic_wand_2.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tbDownload">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>261</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string>Download slot list from controller</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/bullet_arrow_down.png</normaloff>:/icons/bullet_arrow_down.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tbUpload">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>261</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string>Upload slot list to controller</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/bullet_arrow_up.png</normaloff>:/icons/bullet_arrow_up.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionSetAnalogInputSlot">
   <property name="text">
    <string>Analog input</string>
   </property>
  </action>
  <action name="actionSetDiscreteInputSlot">
   <property name="text">
    <string>Discrete input</string>
   </property>
   <property name="toolTip">
    <string>Discrete input</string>
   </property>
  </action>
  <action name="actionSetDiscreteOutputSlot">
   <property name="text">
    <string>Discrete output</string>
   </property>
   <property name="toolTip">
    <string>Discrete output</string>
   </property>
  </action>
  <action name="actionSetDHTxxSlot">
   <property name="text">
    <string>DHTxx sensor</string>
   </property>
   <property name="toolTip">
    <string>DHTxx sensor</string>
   </property>
  </action>
  <action name="actionSetEmptySlot">
   <property name="text">
    <string>Empty slot</string>
   </property>
   <property name="toolTip">
    <string>Empty slot</string>
   </property>
  </action>
  <action name="actionResetIoslots">
   <property name="text">
    <string>Reset slots</string>
   </property>
   <property name="toolTip">
    <string>Reset slots</string>
   </property>
  </action>
  <action name="actionSetDallasTemperatureSlot">
   <property name="text">
    <string>Dallas temperature sensor</string>
   </property>
   <property name="toolTip">
    <string>Dallas temperature sensor</string>
   </property>
  </action>
  <action name="actionSetMHZ19Slot">
   <property name="text">
    <string>MH-Z19 sensor</string>
   </property>
   <property name="toolTip">
    <string>MH-Z19 sensor</string>
   </property>
  </action>
  <action name="actionSetSHT2xSlot">
   <property name="text">
    <string>SHT2x sensor</string>
   </property>
   <property name="toolTip">
    <string>SHT2x sensor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
